src/main/java/bimo/tasks/Task.java

```diff
@@ -0,0 +1,61 @@
+package bimo.tasks;
+
+/**
+ * Creates a Task with completion status and description.
+ */
+public class Task {
+    private boolean status = false;
+    private String details;
+
+    /**
+     * Instantiates a Task object.
+     *
+     * @param details Description of object.
+     */
+    public Task(String details) {
+        this.details = details;
+    }
+
+    /**
+     * Sets task as completed.
+     */
+    public void markCompleted() {
+        this.status = true;
+    }
+
+    /**
+     * Sets task as uncompleted.
+     */
+    public void markUncompleted() {
+        this.status = false;
+    }
+
+    /**
+     * Converts task to string value with description.
+     *
+     * @return String value of task.
+     */
```

Comment:
JavaDoc is really good throughout the entire program, so really good job on that as well!